### PR TITLE
Docs: add Dataviz features to Cloud what's new

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,94 @@ weight: -37
...
+<!-- OSS, Enterprise -->
+<!-- already in on-prem release notes -->
+
+<Release date>
+
+_Available in public preview in Grafana Cloud_
+
+You can now visualize enum data in the Time series and State timeline visualizations. To visualize enum data you must first convert the field to an enum field via the Convert field type transformation.
+
+TODO: Add image / gif / video
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:87]:
```suggestion
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,94 @@ weight: -37
...
+<Release date>
+
+_Available in public preview in Grafana Cloud_
+
+You can now zoom in on the y-axis of your time series visualizations. This is useful when you want to focus on a specific range of values.
+
+To learn more, refer to our [Time series documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/time-series/#TODO).
+
+TODO: Add image / gif / video
+
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:74]:
```suggestion
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 3]
```diff
@@ -18,6 +18,87 @@ weight: -37
...
+- **Generate dashboard save changes summary** - You can now generate a summary of the changes you've made to a dashboard when you save it. This is great for effortlessly tracking the history of a dashboard.
+
+To enable these features, you must first enable the `dashgpt` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/#experimental-feature-toggles). Then install and configure Grafana's LLM app plugin. For more information, refer to the [Grafana LLM app plugin documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/llm-plugin/).
+
+When enabled, look for the **✨ Auto generate** option next to the **Title** and **Description** fields in your panels and dashboards, or when you press the **Save** button.
+
+{{< figure src="/media/docs/grafana/dashboards/auto-generate-description-10-2.gif" max-width="750px" caption="Auto-generate a panel description using AI" >}}
+
+## ## Create interactive buttons in canvas visualizations
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:42]:
```suggestion
## Create interactive buttons in canvas visualizations
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 4]
```diff
@@ -18,6 +18,87 @@ weight: -37
...
+  
+_Available in public preview in Grafana Cloud_
+
+You can now add buttons to your canvas visualizations. Buttons can be configured to call an API endpoint. This pushes Grafana's capabilities to new heights, allowing you to create interactive dashboards that can be used to control external systems.
+
+To learn more, refer to our [Canvas button element documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/canvas/#button).
+
+{{< video-embed src="/media/docs/grafana/2023-20-10-Canvas-Button-Element-Enablement-Video.mp4" max-width="750px" caption="Canvas button element demo" >}}
+
+## ## Zoom in on the y-axis of the time series and candlestick visualizations
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:58]:
```suggestion
## Zoom in on the y-axis of the time series and candlestick visualizations
```

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 5]
```diff
@@ -39,6 +39,87 @@ _Generally available in Grafana Cloud_
...
+
+## Zoom in on the y-axis of the time series and candlestick visualizations
+
+<!-- Nathan Marrs -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem release notes -->
+
+October 24, 2023
+
+_Available in public preview in Grafana Cloud_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:87]: This is what's in the 10.2 What's new
```suggestion
_Generally available in Grafana Cloud_
```

**Review by nmarrs — APPROVED** [review]
🚀

**Pull request merged**